archaius2-api/src/main/java/com/netflix/archaius/api/Property.java

```diff
@@ -44,6 +44,14 @@
  */
 public interface Property<T> extends Supplier<T> {
     /**
+     * Token returned when calling onChange through which change notification can be
+     * unsubscribed.  
+     */
+    interface Subscription {
```

Comment:
doesn't this need to be explicitly public for appropriate visibility?

Reply:
nope